src/vivarium_csu_alzheimers/model_specifications/model_spec.yaml

```diff
@@ -1,4 +1,9 @@
 components:
+    # Causes an error if left empty. Uncomment when you have components. 
```

Comment:
I flipped these due to an issue in VPH I am not sure if it is a bug. I will try and investigate later today.